addons/account/models/account_invoice.py

```diff
@@ -1193,8 +1197,8 @@ def action_move_create(self):
         for inv in self:
             if not inv.journal_id.sequence_id:
                 raise UserError(_('Please define sequence on the journal related to this invoice.'))
-            if not inv.invoice_line_ids:
-                raise UserError(_('Please create some invoice lines.'))
+            if not inv.invoice_line_ids.filtered(lambda line: line.account_id):
+                raise UserError(_('Please create some accountable invoice lines.'))
```

Comment:
While we are at it couldn't we have a real emssage, like "Please create at least one accountable invoice line." ? The 'some' is a bit strange.

Reply:
Changed